Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -219,19 +229,76 @@ public void
         {
             // Arrange
             var head = new Node();
+            var node = head;
 
-            foreach (int i in Enumerable.Range(0, 20))
+            int maxDepth = 10;
+            int iterations = (maxDepth / 2) + 1; // Each iteration adds two levels of depth to the graph
+            foreach (int i in Enumerable.Range(0, iterations))
             {
                 var newHead = new Node();
-                newHead.Children.Add(head);
-                head = newHead;
+                node.Children.Add(newHead);
+                node = newHead;
             }
 
             // Act
-            string result = Formatter.ToString(head);
+            string result = Formatter.ToString(head, new FormattingOptions
+            {
+                MaxDepth = maxDepth
+            });
 
             // Assert
-            result.Should().ContainEquivalentOf("maximum recursion depth");
+            result.Should().ContainEquivalentOf($"maximum recursion depth of {maxDepth}");
```

Comment:
nit: are you being too DRY here when including `maxDepth` from the Arrange phase in the Assert phase?

Reply:
I decided to do it like that because I also include it in the calculations.